project/legislation/baseclass.py

```diff
@@ -0,0 +1,85 @@
+"""
+The base class from which all ProPublica and OpenSecrets functions derive their power,
+from the base client to interacting with theirrespective APIs and/or django databases.
+"""
+
+import .json
+import .logging
+import .httplib2
+#this goes with the congresscritters class
+from django.db import models
+
+class PClient(object):
```

Comment:
Remember to rename this class to something more meaningful. If ProPublica and OpenSecrets APIs have similar structure, create a base class for them both to inherit from.